apps/src/code-studio/components/libraries/LibraryIdImporter.jsx

```diff
@@ -0,0 +1,103 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Radium from 'radium';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import i18n from '@cdo/locale';
+import color from '@cdo/apps/util/color';
+import {
+  DEFAULT_MARGIN,
+  libraryStyles
+} from '@cdo/apps/code-studio/components/libraries/styles';
+
+const styles = {
+  linkBox: {
+    cursor: 'auto',
+    height: '32px',
+    marginBottom: 0,
+    flex: 1,
+    maxWidth: 400
+  },
+  inputParent: {
+    display: 'flex',
+    alignItems: 'baseline'
+  },
+  add: {
+    margin: DEFAULT_MARGIN,
+    color: color.dark_charcoal,
+    borderColor: color.dark_charcoal,
+    ':disabled': {
+      color: color.light_gray,
+      borderColor: color.light_gray,
+      backgroundColor: color.lightest_gray
+    }
+  },
+  error: {
+    color: color.red,
+    textAlign: 'left',
+    margin: DEFAULT_MARGIN,
+    minHeight: 30,
+    whiteSpace: 'pre-wrap',
+    lineHeight: 1
+  }
+};
+
+export class LibraryIdImporter extends React.Component {
+  static propTypes = {
+    addLibraryById: PropTypes.func.isRequired,
+    isLoading: PropTypes.bool,
+    loadLibraryErrored: PropTypes.bool
+  };
+
+  state = {
+    importLibraryId: '',
+    wasClicked: false
```

Comment:
do we need `wasClicked` since we have `loadLibraryErrored`? `loadLibraryErrored === true` should indicate that we display an error